src/cbc_sdk/enterprise_edr/base.py

```diff
@@ -45,6 +45,13 @@ class Summary(UnrefreshableModel):
         urlobject = "/api/investigate/v1/orgs/{}/processes/summary"
 
         def __init__(self, cb, model_unique_id):
+            """
+            Initialize the Process object.
```

Comment:
Technically this is initializing the `Process.Summary` object

Reply:
Alternatively you could say "Initializing the Summary object", depending on what design pattern you want to follow. This also applies in the UBS file.

Reply:
OK